src/sdkEmbed.ts

```diff
@@ -159,28 +159,33 @@ export default class SDKEmbed<TType extends IntentType> {
         }
 
         case "onComplete": {
+          throwIf(
+            !!(this.options?.onComplete && this.options.headless),
+            "onComplete is not supported in headless mode!",
```

Comment:
should we suggest the solution as well? like `please wait for the promise to be resolved or rejected`, wdyt?